src/runtime-rs/crates/resource/src/network/network_pair.rs

```diff
@@ -177,3 +177,85 @@ pub async fn get_link_by_name(
 
     Ok(link::get_link_from_message(msg))
 }
+
+#[cfg(test)]
+mod tests {
+    use scopeguard::defer;
+
+    use super::*;
+    use crate::network::network_model::TC_FILTER_NET_MODEL_STR;
+    use test_utils::skip_if_not_root;
+    use utils::link::net_test_utils::delete_link;
+
+    // this ut tests create_link() and get_link_by_name()
+    #[actix_rt::test]
+    async fn test_utils() {
+        if let Ok((conn, handle, _)) =
+            rtnetlink::new_connection().context("failed to create netlink connection")
+        {
+            let thread_handler = tokio::spawn(conn);
+            defer!({
+                thread_handler.abort();
+            });
+
+            skip_if_not_root!();
```

Comment:
Should this be moved to the first of this test?

Reply:
Fixed